docs/advanced/input_files/input-main.md

```diff
@@ -2128,8 +2138,7 @@ These variables are used to control berry phase and wannier90 interface paramete
 
 - **Type**: String
 - **Description**:
-  step number of time center of Gauss type electric field  
-  amp*cos(2pi*f(t-t0)+phase)exp(-(t-t0)^2/2sigma^2)
+  step number of time center of Gauss type electric fieldamp*cos(2pi*f(t-t0)+phase)exp(-(t-t0)^2/2sigma^2)
```

Comment:
Please explain each variable in the function

Reply:
The format is corrected.